src/kili/project/__init__.py

```diff
@@ -35,6 +35,7 @@ def export(  # pylint: disable=too-many-arguments
         single_file: bool = False,
         disable_tqdm: bool = False,
         log_level: LogLevel = "INFO",
+        download_media: bool = True,
```

Comment:
I would call it `with_assets` too or `download_assets`
In the `assets` method, we were directly refering to assets, here we are dealing with a project. I find it more natural to speak about assets and not media. 
Plus, for me the name should be the same as in the CLI to avoid confusions

Reply:
I agree with you that `download_assets` makes more sense, but we use `download_media` a lot in the repository. wdyt @PierreLeveau

Reply:
I agree with @theodu 's point, `download_assets` is more relevant in the export use case.